applications/nrf5340_audio/app.overlay

```diff
@@ -0,0 +1,91 @@
+/ {
```

Comment:
We need to re-sync on this when you are back. The boardfiles are getting a major overhaul in other PRs. See: https://github.com/zephyrproject-rtos/zephyr/pull/49720